Makefile

```diff
@@ -6,8 +6,10 @@ TIMEOUT_UNIT = 20m
 GOFUMPT=gofumpt
 
 e2e-coverage: ## run e2e tests with coverage
-	@go test -failfast -count=1 -tags=$(E2E_TAG) ./tests -coverpkg=./... -coverprofile /tmp/coverage.out
+	tests/e2e.sh
```

Comment:
Might want to rename this "e2e-prepare.sh" or something but it could be done in a follow-up.